tests/postgres/alltypes_test.go

```diff
@@ -128,6 +129,41 @@ WHERE all_types.uuid = 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11';
 	requireLogged(t, query)
 }
 
+func TestIntervalUpsert(t *testing.T) {
+	stmt := SELECT(Employee.AllColumns).FROM(Employee).
+		WHERE(Employee.EmployeeID.EQ(Int(1)))
+
+	var windy model.Employee
```

Comment:
SELECT seems redundant since we can use RETURNING.

Reply:
Done